workshops/scrapbook_css/README.md

```diff
@@ -0,0 +1,263 @@
+---
+name: Creating a jaw-dropping Scrapbook
+description: A simple introduction to CSS and it’s wonderful quirks!
+author: '@sampoder'
+---
+
+### **Creating a jaw-dropping Scrapbook**
+
+![Jaw dropping GIF](https://media2.giphy.com/media/MuTenSRsJ7TQQ/200.gif)
+
+_^ what's going to happen when people see your Scrapbook_
+
+One of the amazing things about building projects for the web is just how easy it is to make it beautiful! To style web projects, we use a language called CSS. It lets us do everything from changing to background colour to animating text!
+
+<img src="https://cloud-5j06exp7f.vercel.app/screenshot_2020-09-12_at_7.29.11_pm.png" width="380" alt="Sam Poder's Scrapbook Profile">
+
+Today, we’re going to be learning about CSS by customizing a [Scrapbook](https://scrapbook.hackclub.com/) profile (^ like mine above). Scrapbook is a platform where hackers can share photos of what they make or just of their daily life. If you don’t have a profile, log in to the Hack Club Slack, visit the [#scrapbook](https://hackclub.slack.com/archives/C01504DCLVD/) channel, take a photo (or use an old one) and post it! 
+
+Next up, you are going to want to open up the [Scrapbook Customizer](https://scrapbook.hackclub.com/customizer/). Type in your username at the top, and delete any code in the right column. Click **‘Go’**.
+
+We’re now ready to get started!
+
+#### Magical Colours!
+
+To make setting colours easier, Scrapbook uses CSS variables. CSS variables work like variables do in any other programming language. You define the variable and then instead of pasting the value in everywhere you add `var(variable_name)`. This makes it so that if you want to change a theme colour (like we're doing today!) you can do it with one line.
```

Comment:
```suggestion
To make setting colours easier, Scrapbook uses CSS variables. CSS variables work like variables do in any other programming language. You define the variable and then instead of pasting the value in everywhere you add `var(--variable-name)`. This makes it so that if you want to change a theme colour (like we're doing today!) you can do it with one line.
```